engine/classes/Elgg/Application/Database.php

```diff
@@ -3,6 +3,7 @@
 namespace Elgg\Application;
 
 use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Query\QueryBuilder;
```

Comment:
Scrutinizer will say there is a namespace conflict may change this to 
``use Doctrine\DBAL\Query\QueryBuilder as DBALQueryBuilder;``

and than use that name in the function params

Reply:
oops read it wrong